### ISSUE TITLE
Search improvement

### ISSUE THREAD

**Issue by rayman89** [Andu2/FEH-Mass-Simulator#163]
When typing something to search it searches and shows results depending on the first letters of the thing you are searching so for for example if I wanna find "hone cavalry" if I type "ho" i can find it but if i type "one" or "cavalry"  it shows no results. Would be possible to improve on that?

**Comment by jyc8** [comment]
Yes, it's a design choice to search by first letter or part of the whole word. I limited it to first letters because some searches can give too many results, I can change it.
